### DISCUSSION TITLE
Birds with persistent repetitive calls and problems that might cause

### DISCUSSION THREAD

**Discussion by w0mbatian** [mcguirepr89/BirdNET-Pi#559] [Q&A]
Where I live in south eastern Australia we have two birds that have monotonous endless repetitive calls in mating season. They are by common names the Wonga Pidgeon and the Tawny Frogmouth. They both call with a brief whooping sound that is repeated a couple of times per second and they can literally call uninterrupted for hours and hours every day or night for weeks on end.
So that obviously leads to a problem with endless identifications and my first question...what can be done about this if anything?
Secondly in only my first overnight run I am seeing quite e few false identifications caused by frog and cricket calls with similar repetive sounds including one identification with high confidence of a bird which is a bee eater so probably won't be about during the night! 
What if anything can be done about this?
Overall BirdNET-PI works fantastically well from what I have seen in my first two days of operation. It is a credit to the author(s) and I have already got a lot out of using it. Thank you!
Ian

**Comment by ThickKnee** [comment]
Hi. I'm also in SE Aust. Been using it for a couple of months.

I second your praise for the BirdNetPi authors. It's a great tool.

But...like you I'm struggling to find the best way to get the maximum value from it. You can add those two species to the exclusion list. Then they will not appear in the detections, altho the program still does the analysis (I think that's right).

I'm more bothered by mistaken detections with high confidence level.

John.

**Comment by w0mbatian** [comment]
Hi John,
I'm in W Tree north of Buchan near the Snowy River. Good to hear from a local.
I am seeing lots of false identifications at night from the very rhythmic frog chorus we have at the moment. Do you see this?
Last night I deleted around 100 identifications of a Tawny Frogmouth that I could not hear at all. 
I feel that there needs to be some way of dealing with such situations in the program.
And the Wonga pidgeons are just starting to call so there will be a similar problem in the daytime.
Also perhaps the logic in the program could check for whether the identification is logical. For instance that a daylight bird is not identified in the middle of the night.
Ian

**Comment by ThickKnee** [comment]
G'day Ian.
We used BirdNet-Pi at Marlo a month ago. We got the Rainbow Bee Eater too from either frogs or insect calls.
You must put the Frogmouth and Wonga in the Species Exclusion List. Then you will not get any detections for them. We do that if there is a dominant species, such as Magpie or Musk lorikeet.
I think the Frogmouth can also arise from human made noises as can the Boobook and Scrub turkey.
It would be good to share experiences of false positives within the Australian context. You musn't have blind faith in BirdNet as it makes  many false positives. I need to use it more to work out just how it can be a really useful birding tool.
Did you install it by following Patrick's guide or that from Core Electronics? I followed Core's guide.
John.

**Comment by w0mbatian** [comment]
G'day John,
I had just about decided that the Tawny Frogmouth would have to go in the exclusion list. In the last 2 nights I had 462 false positives for it. 
Today I decided to up the minimum confidence to 85% but this evening there are still many  false id's happening.
We have a frog chorus at night here at the moment and that seems to give the algorithm a hard time. Also muffled human voices from a TV also get recognised as a Tawny Frogmouth. I have not heard a real Tawny Frogmouth yet this season.
The Wongas have just started to call around here but I am not getting a silly number for them so far.
I really wonder about the sense of endlessly recording identifications even if they are real. For instance I can get many real id's of a fairy wren for instance but that gives no idea really of the number of them around, just that they are about.
What do you think?
It is really quiet where I live and I think I am getting some really nice recordings. Unfortunately I have satellite internet and it is not feasible for me to upload to BirdWeather so I won't get to share them all with the world.
I followed Patrick's instructions on this site.
Ian

**Reply by ThickKnee** [reply]
Ian
_"get many real id's of a fairy wren for instance but that gives no idea really of the number of them around, just that they are about."_
BirdNet-Pi can't be used for counts. I guess that will always require a human with binoculars. We were getting the fairy wrens misidentified as another wren species. That's our big worry -- if it gives a false positive with a decent confidence level, then how do we know when to trust it.

That said, I think it's still useful, but we haven't yet worked out the best way to use it.

I do love the spectrographs. BirdNet-Pi has made me think more about bird acoustics. The experts do claim that calls are more useful than sightings.

We got two confirmed Lyrebird detections. That's some achievement for it  -- beautiful spectrograph. But then, it's identified kitchen noises as Lyrebird too.

BTW, take great care when playing reference calls from xeno canto or other sources. If BirdNet-Pi hears them it will give you a faultless detection.

**Comment by lloydbayley** [comment]
I have the Variegated Fairywren in my list of excluded species as it was picked up as the crickets at night. I did this almost from the start. Barking Owl, Powerful Owl and Eastern Standard Owl have made it onto there too. Dog barks surprise, surprise for the Barking Owl. There is also a bird I had to exclude as it would be picked up as a siren....could be the ThickKnee...not sure.
